src/socket.c

```diff
@@ -947,7 +947,7 @@ void pthreads_socket_get_last_error(zval *object, zend_bool clear, zval *return_
 	if(threaded->store.sock->error == SUCCESS) {
 		RETURN_FALSE;
 	}
-	ZVAL_LONG(return_value, threaded->store.sock->error);
+	RETURN_LONG(threaded->store.sock->error);
```

Comment:
this will not work as expected, did you mean `RETVAL_LONG`?

Reply:
Yep, you're right. `RETURN_LONG` contains a `return;`. My fault.